### ISSUE TITLE
Rancher server stuck in restart loop

### ISSUE THREAD

**Issue by azelezni** [rancher/rancher#23250]
Rancher server stuck in restart loop.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x240 pc=0x218d12a]

goroutine 23440 [running]:
github.com/rancher/rancher/pkg/controllers/user/alert/watcher.(*NodeWatcher).checkNodeReady(0xc000f7d730, 0xc0074fab28, 0x0)
    /go/src/github.com/rancher/rancher/pkg/controllers/user/alert/watcher/node.go:196 +0x3a
github.com/rancher/rancher/pkg/controllers/user/alert/watcher.(*NodeWatcher).checkNodeCondition(0xc000f7d730, 0xc0074fab28, 0x0)
    /go/src/github.com/rancher/rancher/pkg/controllers/user/alert/watcher/node.go:130 +0x9b
github.com/rancher/rancher/pkg/controllers/user/alert/watcher.(*NodeWatcher).watchRule(0xc000f7d730, 0xc01bc50f98, 0x6fc23ac01)
    /go/src/github.com/rancher/rancher/pkg/controllers/user/alert/watcher/node.go:109 +0x573
github.com/rancher/rancher/pkg/controllers/user/alert/watcher.(*NodeWatcher).watch(0xc000f7d730, 0x421ef20, 0xc00e77bbc0, 0x6fc23ac00)
    /go/src/github.com/rancher/rancher/pkg/controllers/user/alert/watcher/node.go:49 +0xa8
created by github.com/rancher/rancher/pkg/controllers/user/alert/watcher.StartNodeWatcher
    /go/src/github.com/rancher/rancher/pkg/controllers/user/alert/watcher/node.go:44 +0x318
```

**Comment by azelezni** [comment]
[container (2).log](https://github.com/rancher/rancher/files/3694338/container.2.log)
Rancher server logs.

**Comment by superseb** [comment]
Can you provide the full logs with the panic included? Can you provide additional information on how to reproduce? When fixing the panic it helps to validate if its fixed by running the steps to reproduce.

See https://rancher.com/docs/rancher/v2.x/en/contributing/#checklist-for-filing-issues why it helps to provide as much as information possible to help with the investigation.

**Comment by alena1108** [comment]
@azelezni what's the rancher version you are on?

**Comment by alena1108** [comment]
The issue should be fixed in v2.3.0
